jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/TableUtils.kt

```diff
@@ -155,3 +167,21 @@ private fun Component.setSelectionHighlighting(table: JTable, isSelected: Boolea
         background = table.background
     }
 }
+
+private class SpeedSearchHighlighter : Highlighter.HighlightPainter {
+    override fun paint(g: Graphics?, p0: Int, p1: Int, bounds: Shape?, component: JTextComponent?) {
```

Comment:
What does it do on a wrap boundary like

```
aaaaaaa Fo
o
```

and search for foo

Reply:
it doesn't because we wrap on words (woo dodging that problem)

Reply:
There are a lot more things we need to do to fix wrapping, disabled while wrapping. I think it's still worth doing even if it only works for unwrapped because that is the default state.